src/components/popup-menu/popup-menu.scss

```diff
@@ -0,0 +1,38 @@
+$userPanelBackground: $blueSecondary;
+$userPanelPadding: $layoutDefaultPadding;
+
+$include-html: false !default;
+
+@if ($include-html) {
+  .mint-popup-menu {
+    position: relative;
+    width: 100%;
+    padding: $userPanelPadding/3 $userPanelPadding/2;
```

Comment:
I am still confused by how do we chose what parameters go into variables?

Why `width:100%` is not `width: $userPanelWidth`, but `padding:$userPanelPadding/3` and not `padding: $layoutDefaultPadding/3`?